torch/_inductor/codecache.py

```diff
@@ -249,35 +253,151 @@ def code_hash(code):
     )
 
 
-def get_code_path(source_code: Union[str, bytes], ext: str, extra: str, binary=False):
-    if binary:
-        source_code = repr(source_code)
-    basename = code_hash(source_code + extra)
+def serialize_fx_arg(arg: Any):
+    if isinstance(arg, torch.fx.GraphModule):
+        # Special case for fx_graph
+        return arg.print_readable(print_output=False).encode("utf-8")
```

Comment:
readable includes stack traces which you probably don't want to cache on

Reply:
`str(arg)` should be sufficient?

Reply:
ok, changed to that